disnake/player.py

```diff
@@ -578,10 +578,8 @@ async def probe(
                 _log.exception("Fallback probe using '%s' failed", executable)
             else:
                 _log.info("Fallback probe found codec=%s, bitrate=%s", codec, bitrate)
-        else:
-            _log.info("Probe found codec=%s, bitrate=%s", codec, bitrate)
-        finally:
-            return codec, bitrate  # noqa: B012
+        _log.info("Probe found codec=%s, bitrate=%s", codec, bitrate)
+        return codec, bitrate
```

Comment:
The `else` part should've been fine here.

Reply:
https://github.com/DisnakeDev/disnake/pull/1337/commits/25ae68c2487bbd77af87460f89f602cae8ae5f50
This now technically doesn't catch `BaseException`s anymore, though I don't imagine there could be occurring any here that we wouldn't want to re-raise anyway.